jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,18 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+mountpath="$WORKSPACE/RackHD/docker/files/mount"
+retrytimes=5
+while [ ! -d "$mountpath" ]
+do
+    echo "mount is not finished"
+    retrytimes=$(($retrytimes-1))
+    echo "retry : $retrytimes times"
+    sleep 10
+    if [ $retrytimes -eq 0 ]; then
+        break
+    fi
```

Comment:
yes , however   if after 5 times retry , we have no need to wait, the mount opreation should failed ,  if  I  add  if [! -d $ $mountpath]   before chown  operation , it does will not do the chown , but , the result will be same ,script will going on , becuase the failed will not stop the script , so I think we have no need to add if judgement before chown .